packages/lexicon/src/validators/primitives.ts

```diff
@@ -223,25 +229,45 @@ export function string(
 
   // maxGraphemes
   if (typeof def.maxGraphemes === 'number') {
-    if (graphemeLen(value) > def.maxGraphemes) {
-      return {
-        success: false,
-        error: new ValidationError(
-          `${path} must not be longer than ${def.maxGraphemes} graphemes`,
-        ),
+    if (value.length <= def.maxGraphemes) {
+      // If the JavaScript string length is within the maximum limit,
```

Comment:
might be worth mentioning here (and below) that JavaScript string length is UTF-16, just to be super redundant about that, because it is confusing to many devs.